doc/conf.py

```diff
@@ -20,12 +20,12 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
+# sys.path.insert(0, os.path.abspath('.'))
 
```

Comment:
Do you use a automated style check?

Content in this ``conf.py`` was generated by ``sphinx`` configuration and I would prefer to let it stay as it is due to this nature.